### PR TITLE
Update spanish translations

### PR DESCRIPTION
**Pull request by SofLesc**
| Q             | A
| ------------- | ---
| Bug fix?      |no
| New feature?  | no
| BC breaks?    | no
| Deprecations? |no
| Tests pass?   | yes
| Documentation | no
| Translation   | yes
| CHANGELOG.md  | no
| License       | MIT

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.es.yml [hunk 1]
```diff
@@ -420,24 +420,24 @@ quickstart:
...
     new:
         add: 'Añadir'
         placeholder: 'Puedes añadir varias etiquetas, separadas por una coma.'
     rename:
-        # placeholder: 'You can update tag name.'
+        placeholder: 'Puedes actualizar el nombre de la etiqueta.'
 
-# export:
+export:
 #     footer_template: '<div style="text-align:center;"><p>Produced by wallabag with %method%</p><p>Please open <a href="https://github.com/wallabag/wallabag/issues">an issue</a> if you have trouble with the display of this E-Book on your device.</p></div>'
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.es.yml]: Can't you translate that one too?

File: src/Wallabag/CoreBundle/Resources/translations/messages.es.yml [hunk 2]
```diff
@@ -640,13 +640,13 @@ flashes:
...
         notice:
-            added: 'Añadido el usuario "%username%"'
-            updated: 'Actualizado el usuario "%username%"'
-            deleted: 'Eliminado el usuario "%username%"'
+            added: 'El usuario "%username%" ha sido añadido'
+            updated: 'El usuario "%username%" ha sido actualizado'
+            deleted: 'El usuario "%username%" ha sido eliminado'
     site_credential:
         notice:
             # added: 'Site credential for "%host%" added'
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.es.yml]: These one too?

**Comment by SofLesc** [comment]
Hi @j0k3r regarding translation I was wondering which translation should I trust and use as reference. I noticed that in english translation it talks often about `entires` like `unread entries` but on spanish, portuguese and french it talk about `articles` linke `unread articles`. What should I do ?

**Comment by j0k3r** [comment]
I would say: use the english one.
We tend to use _article_ instead of _entries_ in french. I found it better FMPOV.
I can't remember why we decided to use entries instead of articles for english.

Maybe @nicosomb knows?

**Comment by SofLesc** [comment]
I would have used articles for english too. And articles seemed better for spanish/portuguese translations and coherent with the other translations.

I updated `validators.es.yml`

**Comment by nicosomb** [comment]
@j0k3r I can't remember 🤷‍♂️

**Comment by j0k3r** [comment]
Thanks for the work @SofLesc!
Maybe we'll update the english doc to convert entries to articles, we'll discuss it and we'll see.

**Comment by nicosomb** [comment]
Thank you @SofLesc 💪👍

**Pull request merged**